### DISCUSSION TITLE
New theme

### DISCUSSION THREAD

**Discussion by make-github-pseudonymous-again** [tridactyl/tridactyl#4255] [Show and tell]
I wrote [a theme based on `dark`](https://github.com/make-github-pseudonymous-again/dotfiles/blob/dcf5d1a17c522986f47dc13da53f6c476ee867f7/.config/tridactyl/themes/sane.css) for my own use. I'd just like to share it. Provided enthusiasm ensues, maybe we can find a better name for it, have some light version of it, and then add it to the list of themes?

**Comment by bovine3dom** [comment]
If it's better than the current dark theme we can just replace it since people on the whole don't like it (see #1129)

As for a light version maybe it would be better to finish #3292 which would allow themes to be combined easily. From what I remember it is 95% finished :)
